Cookbook/Technical-Documents/SnapshotsTips.md

```diff
@@ -0,0 +1,63 @@
+# Snapshot Testing tips
+
+## What are snapshot tests?
+
+We're testing visual layer of our apps by using snapshots. Snapshots in our case are images of screens and UI components. They are stored in PNG files.
+
+We're using [SnapshotTesting](https://github.com/pointfreeco/swift-snapshot-testing) library from pointfree to record and compare our snapshots.
+
+This kind of tests help us catching any regressions in visual appearance. For example they are very useful when working on a component which might be used on many screens.
+
+## Tips
+
+### Building UI
+
+Until we adopt [SwiftUI previews](https://nshipster.com/swiftui-previews/), snapshots are the fastest way to work iteratively on new UI. 
+Using snapshot tests as feedback mechanism when building UI is much faster than compiling, launching full app, and verifying appearance only then.
```

Comment:
```suggestion
Using snapshot tests as feedback mechanism when building UI is much faster than compiling, launching the full app, reaching the desired page and state, and verifying appearance only then.
```

Reply:
Nice, making it seem even more cumbersome. I like it very much. 😄